main/uart_nic.c

```diff
@@ -65,50 +61,57 @@ static const uint16_t INACTIVE_BEACON_SECONDS = 3600 * 18;
 // pings to the AP?
 static const uint32_t INACTIVE_PACKET_SECONDS = 5;
 
-// intron
-// 0 as uint8_t
-// fw version as uint16_t
-// hw addr data as uint8_t[6]
-#define MSG_DEVINFO 0
-
-// intron
-// 1 as uint8_t
-// link up as bool (uint8_t)
-#define MSG_LINK 1
-
-// intron
-// 2 as uint8_t
-#define MSG_GET_LINK 2
-
-// intron
-// 2 as uint8_t
-// ssid size as uint8_t
-// ssid bytes
-// pass size as uint8_t
-// pass bytes
-#define MSG_CLIENTCONFIG 3
-
-// intron
-// 3 as uint8_t
-// LEN as uint32_t
-// DATA
-#define MSG_PACKET 4
-
-// intron
-// 5 as uint8_t
-// new intron as uint8_t[8]
-#define MSG_INTRON 5
+// Note: Values 1..5 are reserved and must not be used.
```

Comment:
Again, „reserved“…

(Though it would probably not be a big issue if there were reused in some future version).